src/components/common/CopyButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconButton } from '@rmwc/icon-button';
+import React from 'react';
+
+interface CopyButtonProps {
+  text: string;
+  label?: string;
+}
+
+// TODO(davideast): Depend on pkg https://github.com/jaredLunde/react-hook/tree/master/packages/copy#readme
+function useCopy(text: string) {
+  const [copied, setCopied] = React.useState(false);
+  const reset = React.useRef(() => setCopied(false));
+  React.useEffect(() => reset.current, [text]);
```

Comment:
doesn't seem like reset is used anywhere? Safe to drop?

Reply:
This was a port of [`useCopy()`](https://github.com/jaredLunde/react-hook/tree/master/packages/copy). I didn't think we would use our own version but I didn't want to bring in a dependency before getting approval. So ignore this hook unless you want us to use our own.